### ISSUE TITLE
Error parsing quoted targetargs

### ISSUE THREAD

**Issue by NeilCross** [OpenCover/opencover#572]
I'm trying to perform coverage of a DNX app as per [the DNX Support wiki article](https://github.com/OpenCover/opencover/wiki/DNX-Support), and It's appears to be having problems parsing the targetargs if it contains a space character.

`C:\Users\username\.dnx\packages\OpenCover\4.6.519\tools\OpenCover.Console.exe -target:"C:\dnx-clr-win-x86.1.0.0-rc1-update2\dnx-clr-win-x86.1.0.0-beta6\bin\dnx.exe" -targetargs:"--lib c:\projects\project\website\bin\debug\dnx451 c:\projects\project\test\Website.Tests test" -output:coverage.xml -filter:+[Website]*`

Returns an error from opencover: 
`Incorrect Arguments: The argument 'c:\projects\project\website\bin\debug\dnx451' is not recognised
Usage:
    ["]-target:<target application>["]
    [["]-targetdir:<target directory>["]]
    [["]-searchdirs:<additional PDB directory>[;<additional PDB directory>][;<additional PDB directory>]["]]
    [["]-targetargs:<arguments for the target process>["]]
    [-register[:user]]
    ...`

**Comment by molnargab** [comment]
Maybe some extra quotes? There are a couple of options in [Usage#notes-on-spaces-in-arguments](https://github.com/OpenCover/opencover/wiki/Usage#notes-on-spaces-in-arguments).

**Comment by sawilde** [comment]
you have 

    -targetargs:"--lib c:\projects\project\website\bin\debug\dnx451 c:\projects\project\test\Website.Tests test"

but you'll see the recommendation is 

    [["]-targetargs:<arguments for the target process>["]]

so try

    "-targetargs:--lib c:\projects\project\website\bin\debug\dnx451 c:\projects\project\test\Website.Tests test"

**Comment by NeilCross** [comment]
Ok great, with the change provided I've been able to move forward on this.

There was a problem with the command on the wiki page where the test project path needs to be identified with the -p switch (not sure if this was always a requirement), and I had multiple libs to pass in so I needed a new --lib switch for each of them.

I've been able to get it working, and I've updated the wiki page for dnx to address these problems.

Thanks for your help.
